### DISCUSSION TITLE
Does anyone know how to create a MAUI project that only targets Windows and Mac?

### DISCUSSION THREAD

**Discussion by JHeLiu** [dotnet/maui#23458] [Q&A]
Does anyone know how to create a MAUI project that only targets Windows and Mac?

**Comment by MartyIX** [comment]
When you create a new MAUI project, it contains these two lines:

https://github.com/dotnet/maui/blob/9b44c9a26d7dc2de3b75b36853cd689338e33e84/src/Templates/src/templates/maui-mobile/MauiApp.1.csproj#L4-L5

(with `DOTNET_TFM=net8.0`)

So you just remove whatever platform you don't need and your MAUI project should not be built against that platform.
